CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This file is used to list changes made in the last 3 major versions of the postg
 ## v7.1.5 (2019-11-18)
 
 - Allow to install extensions with hyphens, ex: `postgresql_extension '"uuid-ossp"'`
+- Correctly configure postgres-common on ubuntu hosts, fixes #596
```

Comment:
Can you pop this under the `## unreleased` grouping at the top (or make one if it doesn't exist)